py4j-python/src/py4j/tests/java_gateway_test.py

```diff
@@ -1075,6 +1075,14 @@ def testJavaopts(self):
         self.gateway = JavaGateway.launch_gateway(javaopts=["-Xmx64m"])
         self.assertTrue(self.gateway.jvm)
 
+    def testJvmDebugPort(self):
+        from random import randint
+        dport = randint(5000, 5999)
```

Comment:
Using a random int in case tests are run in parallel and we run into port conflicts. I don't think the actual number matters.